iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -129,11 +124,9 @@
 
   <ItemGroup>
     <Compile Include="$(Common)\DefaultWebProxySettings.cs" />
-    <Compile Include="DigitalTwin\**\*.cs" />
```

Comment:
No need to manually include these since they are no longer excluded from the project in the above statement